### ISSUE TITLE
Input Log: Pairs and Quadruplets (Hamiltonian_Anisotropic)

### ISSUE THREAD

**Issue by GPMueller** [spirit-code/spirit#41]
See also #2

**Comment by GPMueller** [comment]
Maybe keep the input pairs and quadruplets somehow:
- read only pairs and quadruplets from file
- give them to the Hamiltonian to save and build the full list
This way the interaction constants could also be changed in the UI later on and the complete set of pairs re-created.

**Comment by GPMueller** [comment]
Depends on #101

**Comment by GPMueller** [comment]
In the branch of e41fbf89dac8f1ea179c11714f3277e40f82126c this will soon be implemented.

**Comment by GPMueller** [comment]
Switched this effort to branch feature-improve-hamiltonians instead. Since 6238182ffbf5f8ff8a9f849c0129061851d618b5, everything is prepared to cleanly log Pairs and Quadruplets.

**Comment by GPMueller** [comment]
Merged into develop with 8569ae558f62578cc6815b3a3c5800e6ca172722.

**Comment by GPMueller** [comment]
Implemented Pairs with 08983f5b63cb6f6b6485c1383762a274650643aa.

**Comment by GPMueller** [comment]
Implemented Quadruplets with 0c44cb06ca588508b05efec7619b1c612d92a9b6.
